client/gutenberg/editor/controller.js

```diff
@@ -142,7 +144,11 @@ export const redirect = async ( context, next ) => {
 	if ( shouldRedirectGutenberg( state, siteId ) ) {
 		const postType = determinePostType( context );
 		const postId = getPostID( context );
-		const url = getGutenbergEditorUrl( state, siteId, postId, postType );
+
+		const url =
+			postId || ! isSiteUsingCoreSiteEditor( state, siteId )
```

Comment:
It seems to me like this will override existing post editor behavior when `postId` is `null`?

Reply:
It should only override the existing post editor when both `postId` is `null` and there is the site editor sticker present.

So if we have the site editor sticker - it will load the site editor when postId is null.
When we don't have the site editor sticker - it will always load the existing post editor.

Reply:
> When we don't have the site editor sticker - it will always load the existing post editor.

That's what I was thinking about. Are there legitimate cases in when `postId` is null and we need to load the post editor (like when creating a new post or similar)?

Reply:
If there are, then that situation is handled just as it was before this change.

Reply:
Oh 🤔 ... you mean with the site editor enabled as well?  Right...  will look into that.

Reply:
Ok, that makes sense.  What do you think about replacing `postId` here with `postType`?  I pushed those changes.  `postType` seems to always be something in the context of launching the existing post editor, while postId is `null` in terms of the new post.